src/handler.py

```diff
@@ -27,8 +26,7 @@
     """
 
 
-async def text_handler(update: Update, context: ContextTypes.DEFAULT_TYPE, con: psycopg2.connect) -> None:
-    _ = context
+async def store_message(update: Update, con: psycopg2.connect) -> None:
```

Comment:
```suggestion
async def store_message(message: Message, con: psycopg2.connect) -> None:
```

1. `from telegram import Message` should work [because `Message` is a public class](https://docs.python-telegram-bot.org/en/v21.10/telegram.message.html#telegram.Message)
2. The check if message is None should move up to the call site.

Reply:
this Message requires message_id and date. It's like forming a new Message object